hello/src/hello.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "hello.h"
+
+char *
+hello(char *str) {
+  int size = strlen(str) + 10;
```

Comment:
+10 ?
It is necessary only 1 byte for null character, isn't it?
https://codezine.jp/article/detail/4644
You're afraid of buffer over flow?